packages/xl-odt-exporter/src/odt/defaultSchema/blocks.tsx

```diff
@@ -212,18 +224,60 @@ export const odtBlockMappingForDefaultSchema: BlockMapping<
 
   checkListItem: (block, exporter) => (
     <TextP>
-      <TextSpan>{block.props.checked ? "☒" : "☐"}</TextSpan>
+      <TextSpan>{block.props.checked ? "☒" : "☐"} </TextSpan>
       {exporter.transformInlineContent(block.content)}
     </TextP>
   ),
 
+  pageBreak: async () => {
+    return <TextP text:style-name="PageBreak" />;
+  },
+
   image: async (block, exporter) => {
-    await exporter.resolveFile(block.props.url);
-    return (
-      <DrawFrame>
-        <DrawImage href={`Pictures/${block.props.url}`} />
-      </DrawFrame>
+    const { Buffer } = await import("buffer");
+    const blob = await exporter.resolveFile(block.props.url);
+    const arrayBuffer = await blob.arrayBuffer();
+    const base64 = Buffer.from(arrayBuffer).toString("base64");
+    //const dimensions = await getImageDimensions(blob);
+    const styleName = createParagraphStyle(
+      exporter as ODTExporter<any, any, any>,
+      block.props
     );
+    const editorWidth = 623;
+    const width = `${(block.props.previewWidth / editorWidth) * 100}%`;
+    const imageFrame = (
+      <TextP text:style-name={block.props.caption ? "Caption" : styleName}>
+        <DrawFrame
+          style:rel-height="scale"
+          style:rel-width={block.props.caption ? "100%" : width}
+          {...(!block.props.caption && {
+            "text:anchor-type": "as-char",
+          })}>
+          <DrawImage
+            xlink:type="simple"
+            xlink:show="embed"
+            xlink:actuate="onLoad">
+            <OfficeBinaryData>{base64}</OfficeBinaryData>
```

Comment:
does word / openoffice also use this base64 embedding when you create images? Or do they create seperate files (resources) in the zip file. If the latter, that would be preferred (stay as close as possible to what other authoring software does)

Reply:
I moved to separate files, which seems to have fixed some Word compatibility issues.